libs/next_gen_ui_langgraph/readme_example.py

```diff
@@ -6,30 +6,248 @@
 from langgraph.prebuilt import create_react_agent
 from next_gen_ui_langgraph import NextGenUILangGraphAgent
 
-movie_toy_story = [
-    {
-        "movie": {
-            "languages": ["English"],
-            "year": 1995,
-            "imdbId": "0114709",
-            "runtime": 81,
-            "imdbRating": 8.3,
-            "movieId": "1",
-            "countries": ["USA"],
-            "imdbVotes": 591836,
-            "title": "Toy Story",
-            "url": "https://themoviedb.org/movie/862",
-            "revenue": 373554033,
-            "tmdbId": "862",
-            "plot": "A cowboy doll is profoundly threatened and jealous when a new spaceman figure supplants him as top toy in a boy's room.",
-            "posterUrl": "https://image.tmdb.org/t/p/w440_and_h660_face/uXDfjJbdP4ijW5hWSBrPrlKpxab.jpg",
-            "released": "2022-11-02",
-            "trailerUrl": "https://www.youtube.com/watch?v=v-PjgYDrg70",
-            "budget": 30000000,
-            "actors": ["Jim Varney", "Tim Allen", "Tom Hanks", "Don Rickles"],
-        },
-    }
-]
+# Movie database with comprehensive data including box office, awards, and weekly performance
+MOVIES_DB = {
+    "toy_story": {
+        "languages": ["English"],
+        "year": 1995,
+        "imdbId": "0114709",
+        "runtime": 81,
+        "imdbRating": 8.3,
+        "movieId": "1",
+        "countries": ["USA"],
+        "imdbVotes": 591836,
+        "title": "Toy Story",
+        "url": "https://themoviedb.org/movie/862",
+        "revenue": 373554033,
+        "tmdbId": "862",
+        "plot": "A cowboy doll is profoundly threatened and jealous when a new spaceman figure supplants him as top toy in a boy's room.",
+        "posterUrl": "https://image.tmdb.org/t/p/w440_and_h660_face/uXDfjJbdP4ijW5hWSBrPrlKpxab.jpg",
+        "released": "1995-11-22",
+        "trailerUrl": "https://www.youtube.com/watch?v=v-PjgYDrg70",
+        "budget": 30000000,
+        "actors": ["Jim Varney", "Tim Allen", "Tom Hanks", "Don Rickles"],
+        "genres": ["Animation", "Adventure", "Family", "Comedy"],
+        "director": "John Lasseter",
+        "productionCompany": "Pixar Animation Studios",
+        "openingWeekend": 29140617,
+        "domesticRevenue": 191796233,
+        "internationalRevenue": 181757800,
+        "profit": 343554033,
+        "roi": 11.45,
+        "audienceScore": 92,
+        "criticScore": 100,
+        "awards": {"wins": 28, "nominations": 24, "oscars": 1},
+        "weeklyBoxOffice": [
+            {"week": 1, "revenue": 29140617},
+            {"week": 2, "revenue": 18923456},
+            {"week": 3, "revenue": 15234567},
+            {"week": 4, "revenue": 12345678},
+        ],
+    },
+    "the_shawshank_redemption": {
+        "languages": ["English"],
+        "year": 1994,
+        "imdbId": "0110912",
+        "runtime": 142,
+        "imdbRating": 9.3,
+        "movieId": "2",
+        "countries": ["USA"],
+        "imdbVotes": 2589173,
```

Comment:
For testing purposes we have these File https://github.com/RedHat-UX/next-gen-ui-agent/blob/main/libs/next_gen_ui_testing/data_set_movies.py
Please move it there and for testing purposes create a new agent (e.g. copy of readme_example) that would be meant for testing purposes.